src/styles/_catalog.scss

```diff
@@ -1,22 +1,28 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(1, $card-width);
+  gap: 48px;
+  justify-content: center;
 
   &__card {
     display: flex;
```

Comment:
I'm not sure, but I am replaced this flex with grid too. So leave it for mentor)